dashboard/config/routes.rb

```diff
@@ -617,6 +617,13 @@
             patch :bulk_update_owners
           end
         end
+        resources :users, only: [] do
+          collection do
+            get 'new', to: 'users#new_user_landing'
```

Comment:
Avoid using `new` as the name for this, as it is a reserved action in Rails resources, typically used to display "an HTML form for creating a new resource": https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions
```suggestion
            get :greeting
```